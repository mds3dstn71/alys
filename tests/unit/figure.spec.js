import { shallowMount } from '@vue/test-utils';

import Figure from '@/components/figure/figure.vue';

test('renders component', () => {
  const src = 'https://alys.js.org/logo.png';
  const caption = 'Alys logo';
  const wrapper = shallowMount(Figure, {
    propsData: {
      src,
      caption,
    },
  });

  expect(wrapper).toMatchSnapshot();
});
