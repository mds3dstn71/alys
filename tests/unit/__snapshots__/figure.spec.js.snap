// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<figure>
  <img src="https://alys.js.org/logo.png" alt="">
  <figcaption>Alys logo</figcaption>
</figure>
`;
